Makefile

```diff
@@ -22,3 +22,7 @@ container: build
 
 container-run: container
 	docker run --name scanner-trivy --rm -d -p 8080:8080 $(IMAGE)
+
+lint:
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOPATH)/bin v1.21.0
+	golangci-lint run -v
```

Comment:
If we installed linter to `.bin`, this has to be changed to `./bin/golangci-lint run -v`.

This will fail the build on master though, as there're some issues detected by the linter. So either we'll append `|| true` and fix the issues in a dedicated PR, or fix in scope of this one. Anyways we cannot merge it with failing build on Travis CI. Do you want to fix the linter issues in this PR as well?

Reply:
I can fix the linter issues in this PR.